exporter/exporterparser/zipkin_test.go

```diff
@@ -142,12 +143,14 @@ func TestZipkinExportersFromYAML_roundtripJSON(t *testing.T) {
 	defer cst.Close()
 
 	config := `
-exporters:
-    zipkin:
-      upload_period: 1ms
-      endpoint: ` + cst.URL
-	tes, _, doneFns, err := ZipkinExportersFromYAML([]byte(config))
-	if err != nil {
+zipkin:
+  upload_period: 1ms
+  endpoint: ` + cst.URL
+	v := viper.New()
```

Comment:
This pattern is quite common throughout the code and most likely might make writing tests warrant reading through viper but also bytes.NewBuffer.

How about let's make an internal helper to do all this parsing?
```go
func ViperFromYAML(yamlBlob []byte) (*viper.Viper, error) {
   v := viper.New()
   v.SetConfigType("yaml")
   if err := v.ReadConfig(bytes.NewBuffer(yamlBlob)); err != nil {
       return nil, err
   }
   return v, nil
}
```